procrastinate/utils.py

```diff
@@ -233,3 +236,27 @@ async def wrapper():
         yield task
     finally:
         task.cancel()
+
+
+def utcnow() -> datetime.datetime:
+    return datetime.datetime.now(datetime.timezone.utc)
+
+
+def aware_datetime(*args, tz_offset: Optional[int] = None) -> datetime.datetime:
```

Comment:
If I'm not mistaken, this function is only used in tests. Would you mind defining it as a test fixture ? (and maybe remove its own tests)

Reply:
It's only used in tests indeed, however, in at least one occurence, it's called at the module level in a test file, meaning I cannot make it only a fixture. I've made it a function and moved it into conftest :)